libc-top-half/musl/src/thread/pthread_mutex_unlock.c

```diff
@@ -30,6 +30,7 @@ int __pthread_mutex_unlock(pthread_mutex_t *m)
 		if (next != &self->robust_list.head) *(volatile void *volatile *)
 			((char *)next - sizeof(void *)) = prev;
 	}
+#ifdef __wasilibc_unmodified_upstream
 	if (type&8) {
```

Comment:
Do you have a map of what bits are what in the mutex type?  What do bit 8 and bit 128 mean?

Reply:
Not sure.

Reply:
In the long run I think it would be good for us to have a clear understand on what those bits mean and what we are expluding from our build.  But its very early days and I'm OK with landing this as is if you like.